ui/desktop/tests/end2end/authentication.spec.js

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable no-undef */
+const { test, expect } = require('@playwright/test');
+const { _electron: electron } = require('playwright');
+const path = require('path');
+
+let electronApp = null;
+const returnScreenshotPath = (fileName) => {
```

Comment:
Thoughts on using `buildScreenshotPath` or `generateScreenshotPath` to avoid `return` in a function name?

Reply:
Done, thanks for the suggestion.